### PR TITLE
Fix issues turned up by opal_show_help() audit

### PR DESCRIPTION
**Pull request by jsquyres**
See individual commit messages.

### CODE REVIEW

File: ompi/mpiext/ftmpi/c/help-mpi-ft.txt [hunk 1]
```diff
@@ -17,26 +18,6 @@ operation, yet you have requested support for fault tolerance.
 When using this component, normal failure free operation is expected;
 However, failures may cause the application to abort, crash or deadlock.
 
-In this framework, the following components are tested to operate under
```
Comment (bosilca) [inline ompi/mpiext/ftmpi/c/help-mpi-ft.txt:20]: We would like to keep these. It is good that right now we do not have any components that fall into these 2 categories, but this does not means we should remove the text.

Comment (jsquyres) [reply ompi/mpiext/ftmpi/c/help-mpi-ft.txt:20]: ✅   I'll put these messages back.

**Review by awlauria — APPROVED** [review]

**Comment by awlauria** [comment]
@jsquyres lgtm. Any reason to not merge?

**Comment by jsquyres** [comment]
Thanks @awlauria; I had forgotten about this one!

**Pull request merged**